syntax = "proto3";

option optimize_for = LITE_RUNTIME;
option java_multiple_files = false;
option java_package = "com.cmsy.majong.mode.proto";

message GateWayMessage {
    int32 userId = 1;
    bytes data = 2;
}